src/app/story/storyrow/storyrow.component.css

```diff
@@ -50,11 +42,77 @@
     margin: 0;
 }
 
+.node-display-name {
+    font-weight: bold;
+    width: 134px;
```

Comment:
Could you please tell me why all these static widths are used?

Reply:
When I use percentages, it takes it as a percent of the parent. When I do a percentage with the parent, for some reason the siblings pipes go all out of wack. I'm still trying to figure out a way to make this work, just the status as of right now.